.circleci/config.yml

```diff
@@ -25,24 +25,28 @@ jobs:
 
         - run:
             name: Get conda running
+            shell: bash -l
             command: |
-              wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh;
-              chmod +x ~/miniconda.sh;
-              ~/miniconda.sh -b -p ~/miniconda;
-              echo "export PATH=~/miniconda/bin:$PATH" >> $BASH_ENV;
+              wget -q https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh
+              chmod +x ~/miniconda.sh
+              ~/miniconda.sh -b -p ~/miniconda
+              conda init bash
+              rm ~/miniconda.sh
 
         - run:
             name: Install dependencies in conda base environment
+            shell: bash -l
             command: |
+              conda activate base
               conda update -n base -c defaults conda
-              pip install numpy scipy pandas json_tricks matplotlib seaborn nibabel coloredlogs python-picard fire
+              echo `which python`
+              echo `which pip`
+              pip install numpy scipy scikit-learn pandas json_tricks matplotlib \
+                          seaborn nibabel coloredlogs python-picard fire
```

Comment:
we should put this in a requirements.txt file

Reply:
yes I guess so